src/cryptocom/exchange/private.py

```diff
@@ -66,9 +67,10 @@ async def get_open_orders(
 
     async def get_trades(
             self, pair: Pair = None, page: int = 0,
-            page_size: int = 200) -> List[PrivateTrade]:
+            page_size: int = 200, end_ts: int= int(time.time() * 1000),
+            start_ts: int = int(time.time() * 1000)-int(86400*1000)) -> List[PrivateTrade]:
```

Comment:
do we need to provide timestamps by default? create a struct TimelineLength: DAY, WEEK MONTH, HOUR etc. as I know they provide only 24 hours or week max of trades